test-e2e/reverse-proxy.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This is a simple reverse proxy that makes sure that any request to localhost:3333 is forwarded to localhost:3000.
```

Comment:
this should also make it easier for devs to contribute without having to install traefik/nginx/caddy or do anything fancy